src/coreclr/inc/clrconfigvalues.h

```diff
@@ -732,6 +722,28 @@ RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableRiscV64Zba,             W("EnableRiscV64
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableRiscV64Zbb,             W("EnableRiscV64Zbb"),          1, "Allows RiscV64 Zbb hardware intrinsics to be disabled")
 #endif
 
+//
+// These are "legacy" ISA enablement knobs that aren't recommended for use anymore
```

Comment:
Are any of these documented in official docs? If they are not, I do not think we need to keep them.

Reply:
`IlcInstructionSet` is explicitly documented in the repo docs as [subject to change without a breaking change notice](https://github.com/dotnet/runtime/blob/main/src/coreclr/nativeaot/docs/optimizing.md).

Reply:
Command line arguments for direct invocation of crossgen2/ilc are not officially documented/supported.

Reply:
Removed the config knobs and instruction-set switches